server/src/main/java/com/octopus/teamcity/opentelemetry/server/TeamCityBuildListener.java

```diff
@@ -291,7 +291,7 @@ private void createBuildStepSpans(SRunningBuild build, String buildName, Tracer
                 }
                 childSpan.setAttribute(PluginConstants.ATTRIBUTE_SERVICE_NAME, blockLogMessage.getBlockType());
                 childSpan.end(finishedDate.getTime(),TimeUnit.MILLISECONDS);
-                Loggers.SERVER.info("Build step span added");
+                Loggers.SERVER.info("Build step span added.");
```

Comment:
Would debug be better here?
```suggestion
                Loggers.SERVER.debug("Build step span added.");
```

Reply:
Updated. Good catch!